src/App.tsx

```diff
@@ -80,6 +81,13 @@ function App() {
     type: 'FeatureCollection',
     features: []
   });
+  // Campus Marker
+  interface CampusCoordination {
+    CampCordLng: number;
+    CampCordLat: number;
+  }
+
+  const [campusCordination, setCampusCordination] = useState<CampusCoordination>({CampCordLng:initialLng, CampCordLat:intitalLat});
```

Comment:
Can we update these variables to `campusCoordinate` and `setCampusCoordinate`?

Reply:
Yes sure